### ISSUE TITLE
Export TS declarations?

### ISSUE THREAD

**Issue by Newbie012** [frigus02/typescript-sql-tagged-template-plugin#27]
Currently, when using the library on a typescript project, I receive the following import error:

```
Could not find a declaration file for module 'typescript-sql-tagged-template-plugin'. '{workspace}/node_modules/typescript-sql-tagged-template-plugin/build/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/typescript-sql-tagged-template-plugin` if it exists or add a new declaration (.d.ts) file containing `declare module 'typescript-sql-tagged-template-plugin';
```

A quick loot at the build artifact shows indeed that there are no declaration files:

![image](https://user-images.githubusercontent.com/10504365/147063954-00f4813e-e4bc-48d9-bbc3-0b59b13ac13c.png)

**Comment by frigus02** [comment]
Hm, yeah. It didn't occur to me that someone might want to use the plugin programmatically. Actually, I can see this could be useful, though.

I think we just have to turn on [`declaration`](https://www.typescriptlang.org/tsconfig#declaration).

**Comment by frigus02** [comment]
I published version 0.2.0, which contains declaration files: https://unpkg.com/browse/typescript-sql-tagged-template-plugin@0.2.0/build/
